app/src/main/java/com/thefuntasty/mvvmsample/data/store/UserStore.kt

```diff
@@ -0,0 +1,21 @@
+package com.thefuntasty.mvvmsample.data.store
+
+import com.jakewharton.rxrelay2.BehaviorRelay
+import com.thefuntasty.mvvmsample.data.model.User
+import io.reactivex.Observable
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class UserStore @Inject constructor() {
+    private val userRelay = BehaviorRelay.createDefault(User.EMPTY)
+
+    fun setUser(user: User) {
+        userRelay.accept(user)
+        // ... optionally persist user
+    }
+
+    fun getUser(): Observable<User> {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/mvvm-android/blob/b65f7b72a86941d478fa7c5532cf990a6c97281f/app/src/main/java/com/thefuntasty/mvvmsample/data/store/UserStore.kt#L18"></span>Functions with exact one statement, the return statement, can be rewritten with ExpressionBodySyntax.</td>
    </tr>
  </tbody>
</table>